src/test/java/teammates/test/cases/search/FeedbackResponseCommentSearchTest.java

```diff
@@ -78,7 +78,8 @@ public void allTests() {
 
         ______TS("success: search for comments; confirms deleted comments are not included in results");
 
-        commentsDb.deleteDocument(frc1I3Q1S2C2);
+        commentsDb.deleteDocument(commentsDb.getFeedbackResponseComment(frc1I3Q1S2C2.courseId, frc1I3Q1S2C2.createdAt,
```

Comment:
Why is this get changed?

Reply:
`frc1I3Q1S2C2` didn't have actual `responseid` needed to delete the document. This was changed after I made added `responseid` as filter in one of the methods of commentsDb.